lib/service-data/service-data.js

```diff
@@ -30,14 +30,30 @@ const external = {}
 
 external.getRuntimeData = getRuntimeData
 
-external.loadServiceData = () => {
+external.loadServiceData = async () => {
   const sourceObjs = external.getServiceSources()
   const schemas = external.getServiceSchemas()
-  return external.getRuntimeData(sourceObjs, schemas)
-    .then(serviceData => {
-      external.setServiceInstances(serviceData)
-      return serviceData
-    })
+
+  // temporary kludge to ensure renamed models are handled
+  if (schemas['definition.namespace']) {
+    process.stdout.write('switching models to namespace')
+    const servicePath = sourceObjs
+      .filter(sourceObj => sourceObj.source === 'service')
+      .map(sourceObj => sourceObj.path)[0]
+    const childProcess = require('child_process')
+    try {
+      childProcess.execSync(`sed -i '' 's/"model"/"namespace"/g' ${servicePath}/**/*.json`)
+    } catch (e) {
+    //
```

Comment:
Should at a minimum log errors?

Reply:
Going to just drop it since it won't work having thought it through - the user the runnner is running under won't have the necessary privs